pkg/activator/net/revision_backends.go

```diff
@@ -60,15 +63,17 @@ const (
 	probeFrequency time.Duration = 200 * time.Millisecond
 )
 
+var emptySet = sets.NewString()
```

Comment:
Are we sure that this isn't mutated?  I'd be more comfortable if we simply inlined this.

Reply:
We never mutate the object once it's set, we only replace it.

Reply:
I changed it, but as code is written now, it was safe :-)